src/srf/ratpoly.cpp

```diff
@@ -13,84 +13,31 @@
 // and convergence should be fast by now.
 #define RATPOLY_EPS (LENGTH_EPS/(1e2))
 
-double SolveSpace::Bernstein(int k, int deg, double t)
+// indexed by [degree][k][exponent]
+static const double bernstein_coeff[4][4][4] = {
```

Comment:
I would even move these inside the respective functions. (as `static const` of course).

Reply:
I moved them inside thinking it would look worse, but now I like it.